public/mix-manifest.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I read the [documentation for Mix on Laravel's site](https://laravel.com/docs/5.7/mix#introduction) but didn't come across an explanation of what a `mix-manifest.json` file is.  Seems like this may be some alternative to a `webpack.mix.js` file?

Could you also explain where the `public/css/app.js` and `public/js/app.css` files came from or how they were generated?

Reply:
Yes, js files and css files under `public/` actually come from `Laravel Mix`, or to say `webpack`. they are compiled files after running `npm run dev`. It compiles js and css files from `resources/assets/` and put them under `public/` according to the file `webpack.mix.js`. Moreover, Laravel uses js and css under `public/`. The file `mix-manifest.json` is produced by `Laravel Mix`, but I don't exactly know the need. You may reference [this issue](https://github.com/JeffreyWay/laravel-mix/issues/580#issuecomment-326840557).

Reply:
I agree with you. These files are not necessarily under source control. A better solution is to generate during deployment in different environments. In addition, if we remove files under `public/` from source control, the installation guide should add extra information for out-of-box instructions.

Reply:
Yes, I agree.  I think modifications to `.travis.yml` and maybe `docker-compose.yml` along with a description in the `README` would satisfy this requirement.